src/persist-client/src/internal/trace.rs

```diff
@@ -823,60 +823,7 @@ impl<T: Timestamp + Lattice> Spine<T> {
 
         // Normal insertion for the batch.
         let index = batch.len().next_power_of_two();
-        self.introduce_batch(Some(batch), usize::cast_from(index.trailing_zeros()), log);
-    }
-
-    /// Apply some amount of effort to trace maintenance.
-    ///
-    /// The units of effort are updates, and the method should be thought of as
-    /// analogous to inserting as many empty updates, where the trace is
-    /// permitted to perform proportionate work.
-    ///
-    /// When this function is called, `effort` must be non-negative
-    #[allow(dead_code)]
-    pub fn exert(&mut self, effort: &isize, merge_reqs: &mut Vec<FueledMergeReq<T>>) {
```

Comment:
Yeah... the only reason that we never store `None` batches in the Spine (that may need to be serialized etc.) is because we never call `exert`, so removing this unblocks the whole thing I think...